var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -49,8 +50,18 @@ def url_for_version(self, version):
             t = 'http://elpa.mpcdf.mpg.de/elpa-{0}.tar.gz'
         return t.format(str(version))
 
-    def setup_environment(self, spack_env, run_env):
+    # override default implementation which returns static lib
```

Comment:
I think I actually have a fix for this. See:
https://github.com/LLNL/spack/pull/3367/files#r105244781
but I'm fine with making it explicit in this package until #3367 is merged.